### PR TITLE
test: Remove npm build requirement for fedora 24

### PR DESCRIPTION
**Pull request by dperpeet**
It's provided by the nodejs package.

### CODE REVIEW

**Comment by dperpeet** [comment]
bot: Image refresh for fedora-24

**Comment by cockpituous** [comment]
Image creation for fedora-24 in process on host32-rack06.
Log: http://fedorapeople.org/groups/cockpit/logs/refresh-fedora-24-ff7e5f837c7434ee05fcece69517a4f93fa1c542/

**Comment by cockpituous** [comment]
Image creation done: https://github.com/cockpituous/cockpit/commits/refresh-fedora-24-ff7e5f837c7434ee05fcece69517a4f93fa1c542

**Comment by stefwalter** [comment]
Seems related:

```
  File "./verify/check-setroubleshoot", line 121, in testTroubleshootAlerts
    b.wait_in_text(row_selector, "you must fix the labels")
```

**Comment by dperpeet** [comment]
> Seems related:
> 
>   File "./verify/check-setroubleshoot", line 121, in testTroubleshootAlerts
>     b.wait_in_text(row_selector, "you must fix the labels")

Added known issue #4281

File: tools/cockpit.spec [hunk 1]
```diff
@@ -71,7 +71,10 @@ BuildRequires: pcp-libs-devel
 BuildRequires: gdb
 
 %if %{defined gitcommit}
+# on fedora 24, nodejs provides npm
+%if 0%{?fedora} > 0 && 0%{?fedora} < 24
```
Comment (petervo) [inline tools/cockpit.spec]: What about rhel and centos? How is npm packaged there?

Comment (dperpeet) [inline tools/cockpit.spec]: > What about rhel and centos? How is npm packaged there?

I'm not sure, but the fedora-24 build failed without the change. The others still work, so I didn't change them.

Comment (stefwalter) [inline tools/cockpit.spec]: Should be fine. I have a branch that removes almost all the gitcommit stuff from cockpit.spec anyway

**Pull request closed without merging**